docs/source/automol/heuristic.md

```diff
@@ -0,0 +1,4 @@
+# automol.heuristic
+```{eval-rst}
```

Comment:
I would like to have all of the `util` functions on the same page. We can do that by putting the `eval-rst` blocks for the `util` submodules directly into `util.md`, below the current `eval-rst` block there:

```
# automol.util 

some words

\```{eval-rst}  (Remove the slashes -- those are to get my back ticks to show up.)
.. automodule:: automol.util
\```

\```{eval-rst}
.. automodule:: automol.util.heuristic
\```

\```{eval-rst}
.. automodule:: automol.util.matrix
\```

Etc.
```